src/main/java/com/example/decafe/Machine.java

```diff
@@ -76,10 +93,13 @@ public void doProgressBarAnimation(Timer productionTimer, ImageView machineImage
                         new KeyValue(machineProgressBar.progressProperty(), 1)
                 )
         );
+
         // Set the maxStatus
         int maxStatus = 12;
+
         // Create the Property that holds the current status count
         IntegerProperty statusCountProperty = new SimpleIntegerProperty(1);
+
         // Create the timeline that loops the statusCount till the maxStatus
```

Comment:
Make generic method for creating KeyFrame, which take duration and keyValue

Reply:
Not necessary in my opinion, we would need the same number of parameters.